mandatory/3-playing-computer.js

```diff
@@ -28,6 +28,10 @@ const f2 = function (a, b) {
 
 console.log(x);
 console.log(a);
+<<<<<<< HEAD
+//console.log(b);
+=======
+>>>>>>> 8a8564fefa1deece60b24d5b10fa131be32d8a80
```

Comment:
Looks like you have a merge conflict you forgot to remove?

This is what you get when there's a merge conflict, and you need to manually resolve it by choose the code you want to keep. You can also remove all of them